bot/exts/moderation/infraction/management.py

```diff
@@ -44,8 +44,10 @@ def infractions_cog(self) -> Infractions:
     # region: Edit infraction commands
 
     @commands.group(name='infraction', aliases=('infr', 'infractions', 'inf', 'i'), invoke_without_command=True)
-    async def infraction_group(self, ctx: Context) -> None:
-        """Infraction manipulation commands."""
+    async def infraction_group(self, ctx: Context, infr_id: int = None) -> None:
```

Comment:
Do we want to make this the correct typing of `t.Optional[int]`?

Reply:
My understanding is that you shouldn't use `Optional` in d.py when it's the last argument. That's what I've been told in the dpy server at least

Reply:
Ah right, forgot that discord.py reads these typings. That's good then

Reply:
see here: 
https://discordpy.readthedocs.io/en/stable/ext/commands/commands.html?highlight=optional#typing-optional

Reply:
That doesn't explain why it can't be the last argument.

Reply:
It simply isn't meant for the last argument, but I think it would work. That said though, what will now happen is that if the `IntConverter` fails instead of a converter error we will continue with `infr_id` set to None (as if it wasn't passed).

Is this acceptable?

Reply:
I see. I suppose that's fine as is then.